pkg/graphql/resolver/k8s.go

```diff
@@ -62,6 +63,46 @@ func parseArgs(args SSARArgs) *auth.ResourceAttributes {
 	return &attr
 }
 
+type SelfSubjectReview struct {
```

Comment:
Add a comment explaining that this is a copy of the `authenticationv1` struct from kubernetes so that we can serialize SelfSubjectReview.Status.UserInfo.Extra as a string compared to a map, which we're unable to work with in graphql